packages/MSBot/src/msbot-clone-services.ts

```diff
@@ -240,6 +245,8 @@ async function processConfiguration(): Promise<void> {
             if (!args.force) {
```

Comment:
If force true, do we still remove the orphaned bot file?

Reply:
Or should we remove it?

Reply:
If user specified the force flat, then we will just go ahead with cloning without prompting user to confirm. There is no need to remove the bot file in that case.